src/overlays/actors/ovl_Boss_Goma/z_boss_goma.h

```diff
@@ -6,11 +6,61 @@
 
 struct BossGoma;
 
+typedef void(*BossGomaActionFunc)(struct BossGoma*, GlobalContext*);
+
 typedef struct BossGoma {
     /* 0x0000 */ Actor actor;
-    /* 0x014C */ char unk_14C[0x70];
+    /* 0x014C */ SkelAnime skelanime;
+    /* 0x0190 */ BossGomaActionFunc actionFunc;
+    /* 0x0194 */ s16 frameCount; // also used as a timer
+    /* 0x0196 */ s16 patienceTimer; // when non-0: walk towards player, can be stunned
+    /* 0x0198 */ s16 eyeLidBottomRotX;
+    /* 0x019A */ s16 eyeLidTopRotX;
+    /* 0x019C */ s16 eyeClosedTimer; // also used as a second invincibility frames source
+    /* 0x019E */ s16 eyeIrisRotX;
+    /* 0x01A0 */ s16 eyeIrisRotY;
+    /* 0x01A2 */ s16 unusedTimer;
+    /* 0x01A4 */ s16 childrenGohmaState[3]; // 0 not spawned, 1 spawned, -1 dead (-1 set by child gohma)
+    /* 0x01AA */ s16 tailLimbsScaleTimers[4];
+    /* 0x01B2 */ s16 spawnGohmasActionTimer;
+    /* 0x01B4 */ s16 eyeState;
+    /* 0x01B6 */ s16 doNotMoveThisFrame;
+    /* 0x01B8 */ s16 visualState;
+    /* 0x01BA */ s16 invincibilityFrames;
     /* 0x01BC */ s16 subCameraId;
-    /* 0x01BE */ char unk_1BE[0x95E];
+    /* 0x01BE */ s16 disableGameplayLogic;
+    /* 0x01C0 */ s16 decayingProgress; // when defeated, textures are progressively cleared
+    /* 0x01C2 */ s16 noBackfaceCulling;
+    /* 0x01C4 */ s16 blinkTimer;
+    /* 0x01C6 */ s16 lookedAtFrames;
+    /* 0x01C8 */ char unused_1C8[0x8];
```

Comment:
I'm not particularly against naming these `unused_`, but imo it would be better to keep them consistent with the rest of the codebase (for now at least)
```suggestion
    /* 0x01C8 */ char unk_1C8[0x8];
```